generators/run/templates/run

```diff
@@ -266,7 +266,7 @@ case $run_command in
         fi
 
         # Run the serve container, publishing the port, and detaching if required
-        <% if (django) { %>django_run "${project}-serve" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach}"
+        <% if (django) { %>django_run "${project}-serve" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach} ${run_serve_docker_opts}"
         <% } else if (jekyll) { %>bundler_run "${project}-serve" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach}" jekyll serve -P ${PORT} -H 0.0.0.0
         <% } else { %>node_run "${project}-serve" "--env PORT=${PORT} --publish ${PORT}:${PORT} ${detach}" yarn run serve
```

Comment:
Actually need `${run_serve_docker_opts}` on the standard line more than the django one but preferably consistently on all of them. 
:)